lib/smart_proxy_openscap/openscap_lib.rb

```diff
@@ -178,14 +178,12 @@ def forward_arf_file(foreman_api_path, arf_file_path)
       begin
         data = File.read(arf_file_path)
         response = send_request(foreman_api_path, data)
-        # Raise an HTTP error if the response is not 2xx (success).
-        response.value
```

Comment:
response value should raise error if not success. See http://ruby-doc.org/stdlib-1.9.3/libdoc/net/http/rdoc/Net/HTTPResponse.html#method-i-value